src/index.ts

```diff
@@ -7,6 +7,16 @@ import createApp from './app';
 import connectDB from './db';
 import { logger } from './configs/logger';
 
+process.on('uncaughtException', (error: Error) => {
```

Comment:
move on bottom on file, under hood all node.js listeners are added in `process.nextTick` so it no difference where you place such  things (ofcorse  taking in account lexical scope)